integration_tests/src/main/python/conditionals_test.py

```diff
@@ -40,11 +40,13 @@
 if_nested_gens = if_array_gens_sample + if_struct_gens_sample
 
 @pytest.mark.parametrize('data_gen', all_gens + if_nested_gens + decimal_128_gens_no_neg, ids=idfn)
-def test_if_else(data_gen):
+@pytest.mark.parametrize('const_bool', [True, False, None])
```

Comment:
It would be good to test not just with all true, all false, and all random values but also all-null values.

Reply:
Added it.